src/Admin/SiteHealth.php

```diff
@@ -0,0 +1,367 @@
+	 */
+	public function curl_multi_functions() {
+		$undefined_curl_functions = array_filter(
+			[
+				'curl_multi_add_handle',
+				'curl_multi_exec',
+				'curl_multi_init',
+			],
+			static function( $function_name ) {
+				return ! function_exists( $function_name );
+			}
+		);
+
+		$data = [
+			'badge'       => [
+				'label' => esc_html__( 'AMP', 'amp' ),
+				'color' => $undefined_curl_functions ? 'orange' : 'green',
+			],
+			'description' => esc_html__( 'The AMP plugin performs better when these functions are available.', 'amp' ),
+			'actions'     => sprintf(
+				'<p><a href="%1$s" target="_blank" rel="noopener noreferrer">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a></p>',
+				'https://www.php.net/manual/book.curl.php',
+				esc_html__( 'Learn more about these functions', 'amp' ),
+				/* translators: The accessibility text. */
+				esc_html__( '(opens in a new tab)', 'amp' )
+			),
+			'test'        => 'curl_multi_functions',
+		];
+
+		if ( $undefined_curl_functions ) {
+			return array_merge(
+				$data,
+				[
+					'status'      => 'recommended',
+					'label'       => esc_html(
+						_n(
+							'A curl_multi_* function is not defined',
+							'Some curl_multi_* functions are not defined',
+							count( $undefined_curl_functions ),
+							'amp'
+						)
+					),
+					'description' => wp_kses(
+						sprintf(
+							/* translators: %s: the name(s) of the curl_multi_* PHP function(s) */
+							_n(
+								'The following curl_multi_* function is not defined: %s. The AMP plugin performs better when this function is available.',
+								'The following curl_multi_* functions are not defined: %s. The AMP plugin performs better when these functions are available.',
+								count( $undefined_curl_functions ),
+								'amp'
+							),
+							implode(
+								', ',
+								array_map(
+									static function( $function_name ) {
+										return sprintf( '<code>%s()</code>', $function_name );
+									},
+									$undefined_curl_functions
+								)
+							)
+						),
+						[ 'code' => [] ]
+					),
+				]
+			);
+		} else {
+			return array_merge(
+				$data,
+				[
+					'status' => 'good',
+					'label'  => esc_html__( 'The curl_multi_* functions are defined.', 'amp' ),
+				]
+			);
+		}
+	}
+
+	/**
+	 * Gets the test result data for whether the proper ICU version is available.
+	 *
+	 * @return array The test data.
+	 */
+	public function icu_version() {
+		$icu_version       = defined( 'INTL_ICU_VERSION' ) ? (float) INTL_ICU_VERSION : null;
+		$minimum_version   = 65;
+		$is_proper_version = $icu_version >= $minimum_version;
+
+		$data = [
+			'badge'       => [
+				'label' => esc_html__( 'AMP', 'amp' ),
+				'color' => $is_proper_version ? 'green' : 'orange',
+			],
+			'description' => sprintf(
+				esc_html__( 'The version of ICU can affect how the intl extension runs. The minimum recommended version of ICU is %s.', 'amp' ),
```

Comment:
Missing `/* translators: ... */` comment in the preceding line.

Reply:
Good point, https://github.com/ampproject/amp-wp/pull/3886/commits/3e95814eebf88dcc52840c4fed441ce1250877f9 adds the translators comment.